### DISCUSSION TITLE
namecheap

### DISCUSSION THREAD

**Discussion by tiru02** [community/community#141852] [GitHub Education]
### Hi everyone,

i was using GitHub developer pack and tried to register free domain in Namecheap and then i realized that whoever the students not studying in US,UK,CANADA and AUSTRALIA are not eligible for this offer.
This is not mentioned in GitHub student developer pack.
![image](https://github.com/user-attachments/assets/0b181935-8555-4b26-b10e-82038e93efae)

**Comment by flame-0** [comment]
https://github.com/orgs/community/discussions/103509
